packages/pigeon/test/kotlin_generator_test.dart

```diff
@@ -126,6 +126,14 @@ void main() {
     expect(code, contains('interface Api'));
     expect(code, contains('fun doSomething(input: Input): Output'));
     expect(code, contains('channel.setMessageHandler'));
+    expect(code, contains('''
+            }
+            reply.reply(wrapped)
+          }
+        } else {
+          channel.setMessageHandler(null)
+        }
+    '''));
```

Comment:
While this test does check to make sure this specific issue doesn't occur again, I'm not entirely sure this is useful. I'll defer to @stuartmorgan or @gaaclarke as to whether this should be removed, altered, or supplemented though.

Reply:
tks